pkg/git/git.go

```diff
@@ -32,6 +33,62 @@ func GetGitVersion() string {
 	return "Unknown"
 }
 
+// GetDefaultBranch detects and returns the default branch name (main, master, etc.)
+func GetDefaultBranch() (string, error) {
```

Comment:
I'll be testing if either of these commands would work on the server side of a Git server.

Reply:
Works on the server side of a Git server 👍